nservicebus/azure/azure-storage-persistence.md

```diff
@@ -45,18 +45,18 @@ snippet:AzurePersistenceFromAppConfig
 
 The following settings are available for changing the behavior of saga persistence through the `AzureSagaPersisterConfig`section:
 
-- `ConnectionString`: Allows you to set the connectionstring to the storage account for storing saga information, defaults to `UseDevelopmentStorage=true`
+- `ConnectionString`: Allows you to set the connectionstring to the storage account for storing saga information, defaults to `UseDevelopmentStorage=true` in Versions 6 and less, and defaults to `null` in Versions 7 and up.
```

Comment:
Also avoid language with: you, we, etc. See: https://github.com/particular/docs.particular.net#language-preferences